### DISCUSSION TITLE
Error on function requestMamMessages

### DISCUSSION THREAD

**Discussion by jennifercardenasm** [vavadiyahiren/xmpp_plugin#85] [Q&A]
Hi!!

I am using the method flutterXmpp.requestMamMessages(), but unfortunally is given error in every  messages  that is recovery. Another issue is on messages with emojis, which are missed on the recovery message, 

This is a piece of log where a piece of message are recovery:
D/SMACK   ( 8314): RECV (0): <message to="prueba3@pruebasserver/Android"><result xmlns="urn:xmpp:mam:1" queryid="ecb70fc6-fc10-4bbd-aed4-a23ef8de06b4" id="311"><forwarded xmlns="urn:xmpp:forward:0"><delay xmlns="urn:xmpp:delay" stamp="2023-11-08T06:16:42.574Z"/><message xmlns="jabber:client" to="prueba1@pruebasserver" id="1699424202178" type="chat" from="prueba3@pruebasserver/Android"><body>holi</body><TIME xmlns="urn:xmpp:time"><ts>1699424202181</ts></TIME><request xmlns="urn:xmpp:receipts"></request></message></forwarded></result></message>
D/flutter_xmpp( 8314): Received Message <message xmlns='jabber:client' to='prueba1@pruebasserver' from='prueba3@pruebasserver/Android' id='1699424202178' type='chat'><body>holi</body><TIME xmlns='urn:xmpp:time' xmlns:stream='http://etherx.jabber.org/streams'><ts xmlns:stream='http://etherx.jabber.org/streams'>1699424202181</ts></TIME><request xmlns='urn:xmpp:receipts'/></message>
D/TAG     ( 8314):  RECEIVE_MESSAGE-->> {chatStateType=, customText=, from=prueba3@pruebasserver/Android, senderJid=prueba3@pruebasserver/Android, delayTime=0, id=1699424202178, time=1699424202181, type=Message, body=holi, msgtype=chat}

I/flutter ( 8314): [D]  ~~>onChatMessage   {customText: , from: prueba1@pruebasserver/DESKTOP-FVFIGLA, senderJid: prueba1@pruebasserver/DESKTOP-FVFIGLA, time: 0, id: 34FQS-2791, type: chatstate, body: k, msgtype: chat, bubbleType: , mediaURL: , isReadSent: 0, delayTime: 0, chatStateType: active}
**_E/AbstractXMPPConnection( 8314): Exception in packet listener
E/AbstractXMPPConnection( 8314): java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.String org.jxmpp.jid.Jid.toString()' on a null object reference
E/AbstractXMPPConnection( 8314): 	at org.xrstudio.xmpp.flutter_xmpp.Utils.Utils.broadcastMessageToFlutter(Utils.java:173)
E/AbstractXMPPConnection( 8314): 	at org.xrstudio.xmpp.flutter_xmpp.listner.MessageListener.processStanza(MessageListener.java:22)
E/AbstractXMPPConnection( 8314): 	at org.jivesoftware.smack.AbstractXMPPConnection$7.run(AbstractXMPPConnection.java:1271)
E/AbstractXMPPConnection( 8314): 	at org.jivesoftware.smack.AsyncButOrdered$Handler.run(AsyncButOrdered.java:135)
E/AbstractXMPPConnection( 8314): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
E/AbstractXMPPConnection( 8314): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
E/AbstractXMPPConnection( 8314): 	at java.lang.Thread.run(Thread.java:1012)_**

The error seems to be on the flutter_xmpp.Utils.Utils class on the method broadcastMessageToFlutter on the line 173, as the error mentioned before

Thanks in advance for  your help
best regards
Jenny

**Comment by vavadiyahiren** [comment]
This must have been fixed in latest version can you please check it with the latest version and if it is still there we can discuss.
